tests/fe/local_sparsity_01.output

```diff
@@ -0,0 +1,36 @@
+
+DEAL::FE_Q_iso_Q1<1>(2):
+X.X
+.XX
+XXX
+
+DEAL::FESystem<1>[FE_DGQ<1>(1)-FE_Q_iso_Q1<1>(2)^2]:
+XXXXXXXX
```

Comment:
If we can trust the serial CI tester, we should have
```suggestion
XX..XXXX
```
and a few more changes in lines 9, 10, 11, 13 at least (this is due to the renumbering you now take into account).